cmd/podman/ioprojectatomicpodman/io.projectatomic.podman.varlink

```diff
@@ -0,0 +1,64 @@
+# Podman Service Interface
+interface io.projectatomic.podman
+
+type Version (
+  version: string,
+  go_version: string,
+  git_commit: string,
+  built: int,
+  os_arch: string
+)
+
+type NotImplemented (
+    comment: string
+)
+
+type StringResponse (
+    message: string
+)
+
+# System
+method Ping() -> (ping: StringResponse)
+method GetVersion() -> (version: Version)
+
+# Containers
```

Comment:
Missing CommitContainer()

Reply:
It is under the image side of things ...